core/docs/spark-profiling-tool.md

```diff
@@ -29,24 +29,26 @@ or can be found in the location specified by `spark.eventLog.dir`. See the
 [Apache Spark Monitoring](http://spark.apache.org/docs/latest/monitoring.html) documentation for
 more information.
 
-### Step 1 Download the tools jar and Apache Spark 3 distribution
-The Profiling tool requires the Spark 3.x jars to be able to run but do not need an Apache Spark run time. 
+### Step 1a: Download the tools jar
+- Download the Spark RAPIDS tools jar from [Maven repository](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark-tools_2.12/22.12.0/)
+
+If you want to compile the jars, please refer to the instructions [here](./spark-qualification-tool.md#How-to-compile-the-tools-jar).
```

Comment:
Shouldn't we also point to the index page ?

Reply:
I'm not sure what you mean.  The compile steps are a section in the qual tool doc.